src/layers/core/polygon-layer/polygon-layer-64-vertex.glsl

```diff
@@ -0,0 +1,72 @@
+// Copyright (c) 2016 Uber Technologies, Inc.
+//
+// Permission is hereby granted, free of charge, to any person obtaining a copy
+// of this software and associated documentation files (the "Software"), to deal
+// in the Software without restriction, including without limitation the rights
+// to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+// copies of the Software, and to permit persons to whom the Software is
+// furnished to do so, subject to the following conditions:
+//
+// The above copyright notice and this permission notice shall be included in
+// all copies or substantial portions of the Software.
+//
+// THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+// IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+// FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+// AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+// LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+// OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+// THE SOFTWARE.
+#define SHADER_NAME polygon-layer-vertex-shader
+
+attribute vec3 positions;
+attribute vec2 positions64xyLow;
```

Comment:
Have you separated into high and low attributes? That's should be a big step towards unification.

In that case I think we could simplify naming and just call the attributes:

 `positions` and `positions64` (or `positions64xy` if we want to emphasize it is of a different length)

My thinking is that the fact that the attribute is named `64` implicitly means that it contains the additional precision for a corresponding 32 bit attribute, and eventually all our layers follow this convention.